### DISCUSSION TITLE
How to connect external smtp brevo for in email expiry cert reminder?

### DISCUSSION THREAD

**Discussion by syamimirfan** [Keyfactor/ejbca-ce#457] [Q&A]
Hi everyone,

I'm working on implementing an email expiry certificate reminder feature and am looking to connect to an external SMTP service, specifically Brevo. I'm facing challenges in the setup process and have a few questions:

1. Could someone share insights or steps on how to configure our application to use Brevo as the SMTP server for sending certificate expiry reminders?
2. Are there any specific settings or configurations we need to be aware of when integrating with Brevo's SMTP service?

Thank you in advance for your assistance. Any guidance or pointers would be greatly appreciated!

**Comment by primetomas** [comment]
I added a sample command line to the documentation at DockerHub, how to use Gmail SMTP server.
https://hub.docker.com/r/keyfactor/ejbca-ce
I don't know about Brevo, but you should get documentation from them similar to how Google documents how to use their SMTP server.
